runtime/near-vm-runner/src/utils.rs

```diff
@@ -0,0 +1,9 @@
+use std::hash::{Hash, Hasher};
+
+pub(crate) fn stable_hash<T: Hash>(value: T) -> u64 {
+    // Need some kind of stable hash algorithm, previously was using SipHasher via near-stable-hasher
```

Comment:
I’m pretty confident we can live without a stable hashing algorithm for this use-case, and for all other use-cases in this PR. It doesn't even need to be a strong hash.

Reply:
Hmm just so I understand, do you suggest just using DefaultHasher altogether here or just fixing up the comment to say it’s only for backwards-compat and can be reevaluated?

Reply:
I think just removing the comment is fine, but changing to use to `DefaultHasher` or even hand-rolling a shift-xor-add hash here wouldn’t be particularly terrible either.